target/linux/realtek/dts-5.10/rtl8380_netgear_gs308t-v1.dts

```diff
@@ -1,8 +1,34 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 #include "rtl8380_netgear_gigabit_3xx.dtsi"
+#include <dt-bindings/leds/common.h>
```

Comment:
Minor nitpick, but could you add an empty line between the local include and the system file include?